internal/cmd/deployorder.go

```diff
@@ -0,0 +1,128 @@
+package cmd
+
+import (
+	"fmt"
+	"slices"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	"github.com/simplesurance/dependencies-tool/internal/deps"
+)
+
+const deployOrderShortHelp = "Generate a deployment order."
+
+var descrDependencyFileNames = strings.TrimSpace(`
+Dependency definition files are discovered by searching in all child directories
+of PATH.
+Files that match the following names are discovered, only the first found per
+directory is parsed, the preference order is:
+
+  1. .deps-<ENVIRONMENT>-<REGION>.toml
+  2. .deps-<ENVIRONMENT>.toml
+  2. .deps.toml
+`)
+
+var deployOrderLongHelp = deployOrderShortHelp + "\n\n" + strings.TrimSpace(`
+Positional Arguments:
```

Comment:
why the option to use only positional arguments rather than named arguments?

Reply:
Not sure if I understand the question correctly.
Do you mean why path, region, environment, apps are positional arguments instead of named arguments?

Reply:
It follows the usual cli convention that required arguments are positional, named arguments are always optional.
It makes it easier and more straight forward to use to follow the common approach.
It's also requires less typing, but that is a minor thing.